c2corg_ui/static/js/documentediting.js

```diff
@@ -374,6 +389,25 @@ app.DocumentEditingController.prototype.handleMapFeatureChange_ = function(
 
 
 /**
+ * It doesn't associate, it just pushes it into associations array.
+ * @private
+ */
+app.DocumentEditingController.prototype.pushDocToAssociations_ = function() {
```

Comment:
You should rather use the ``apiService`` https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/static/js/apiservice.js